guide/src/setting_up_rust_on_cloud_shell.md

```diff
@@ -0,0 +1,79 @@
+# Setting up Rust on Cloud Shell
+
+Cloud Shell is a great environment to run small examples and tests.
+
+## Start up Cloud Shell
+
+1. Open <https://shell.cloud.google.com> to start a new shell.
+
+1. Select a project.
+
+## Configure Rust
+
+1. [Cloud Shell] comes with [rustup] pre-installed. You can use it to install and configure the default version of Rust:
+
+   ```shell
+   rustup default stable
+   ```
+
+1. Confirm that you have the most recent version of Rust installed:
+
+   ```shell
+   cargo --version
+   ```
+
+## Install Rust client libraries in Cloud Shell
+
+1. Create a new Rust project:
+
+   ```shell
+   cargo new my-project
+   ```
+
+1. Change your directory to the new project:
+
+   ```shell
+   cd my-project
+   ```
+
+1. Add the [Secret Manager] client library to the new project
+
+   ```shell
+   cargo add gcp-sdk-secretmanager-v1 --features unstable-stream
+   ```
+
+1. Add the [tokio] crate to the new project
+
+   ```shell
+   cargo add tokio --features macros
+   ```
+
+1. Edit your project to use the Secret Manager client library:
+
+   ```shell
+   cat >src/main.rs <<_EOF_
+   #[tokio::main]
```

Comment:
nit: Could this be written in rust and included with an anchor similar to what you did in the init client guide? (Same goes for the other inlined code)

Reply:
I like that idea.  The obvious thing renders all wrong:

```
1. Bla bla

    ```rust,ignore,noplayground
    {{#include ../samples/src/bin/getting_started.rs:all}}
    ```
```

Renders to:

![image](https://github.com/user-attachments/assets/f3a78199-2e27-49ef-a8d4-f4d6cd88ff2e)


What I did breaks the itemized list.  I think the tradeoff makes sense (the code is compiled and looks nicer), but let me know what you think.

![image](https://github.com/user-attachments/assets/337b9cb4-40e6-4659-a8d2-55eed3109bc9)

Reply:
> What I did breaks the itemized list. I think the tradeoff makes sense (the code is compiled and looks nicer), but let me know what you think.

LGTM